resolve/RandSketchingFWHT.cpp

```diff
@@ -62,6 +86,18 @@ namespace ReSolve
   }
 
   // Setup the parameters, sampling matrices, permuations, etc
+  /** 
+   * @brief Sketching method setup. This function allocated P(erm), D (diagonal scaling matrix) andpopulates them and  allocates a bunch of other auxiliary arrays.
+   *
+   *
```

Comment:
Separate brief from long:
```suggestion
  /** 
   * @brief Sketching method setup.
   *
   * This function allocated P(erm), D (diagonal scaling matrix) andpopulates them and  allocates a bunch of other auxiliary arrays.
   *
   *
```